sips/sip-108.md

```diff
@@ -0,0 +1,68 @@
+
+## Motivation
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the SIP proposes changing how something is calculated, you must address *why* the current calculation is inaccurate or wrong. This is not the place to describe how the SIP will address the issue!-->
+
+The debt pool is still heavily skewed toward sBTC and sETH, despite active rewards programs for holding iBTC and iETH. In the context of a bull market, this skew represents a significant risk to SNX stakers.
+
+## Specification
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+
+The design of Shorting Rewards is structurally identical to that of the current Staking Rewards contracts. The only difference is that instead of staking tokens, positions are created automatically when users open shorts.
+
+### Rationale
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+The Synthetix Staking Rewards contract has become widely used across DEFI. The underlying structure is well understood by the community.
+
+### Technical Specification
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
+
+A new contract `ShortingRewards.sol` which is a modified version of `StakingRewards.sol`. It shares the same underlying logic for determining reward amounts. Where it differs is that there is no staking token. Instead, it stores a reference to `CollateralShort.sol` and allows this contract to enrol and withdraw users.
+
+Two instances of `ShortingRewards.sol` will be deployed. The first instance will manage rewards for sBTC shorts and the second will manage rewards for sETH shorts.
+
+### Test Cases
+<!--Test cases for an implementation are mandatory for SIPs but can be included with the implementation..-->
+
+Test cases are included with the implementation.
+
+### Configurable Values (Via SCCP)
+
```

Comment:
@mjs-12 

Are there any SCCP values that could be configurable for the ShortingRewards that we can cover here for future SCCP's?